cmd/terramate/e2etests/general_test.go

```diff
@@ -248,10 +247,10 @@ func TestDefaultBaseRefInMain(t *testing.T) {
 	git.Commit("all")
 	git.Push("main")
 
-	// main uses HEAD^1 as default baseRef.
+	// main uses HEAD^ as default baseRef.
 	want := runExpected{
-		Stdout:       stack.RelPath() + "\n",
-		IgnoreStderr: true,
+		Stdout: stack.RelPath() + "\n",
+		//IgnoreStderr: true,
```

Comment:
why is this commented instead of deleted if it is not necessary anymore ?

Reply:
thanks, forgot to delete.